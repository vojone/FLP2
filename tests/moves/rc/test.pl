:- consult('../../../cube').

:- read_cube(C, _), move_r_ccw(C, _, _, RC), write_cube(RC).
