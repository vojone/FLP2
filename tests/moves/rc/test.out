123
453
782
443 369 122 333
116 258 233 444
119 147 333 444
664
661
661
